modules.py

```diff
@@ -0,0 +1,71 @@
+import time
+class Product:
+    __id = ''
+    __name = ''
+    __price = 0.0
+    __imageId = ''
+    __description = ''
+    __reviewsList = []
+    
+    def __init__(self, id, name, price):
+        self.__id = id
+	self.__name = name
+	self.__price = price
+
+    def getId(self):
+        return self.__id
+
+    def setId(self, id):
+        self.__id = id
+
+    def getName(self):
+        return self.__name
+
+    def setName(self, name):
+        self.__name = name
+
+    def getPrice(self):
+        return self.__price
+
+    def setPrice(self, price):
+        self.__price = price
+
+    def getImageId(self):
+        return self.__imageId
+
+    def setImageId(self, imageId):
+        self.__ImageId = imageId
+
+    def getDescription(self):
+        return self.__description
+
+    def setDescription(self, description):
+        self.__description = description
+
+    def getReviewsList(self):
+        return self.__reviewsList
+
+    def setReviewsList(self, reviewsList):
```

Comment:
Do you think this method is necessary? Maybe there is a case when we can use it, but I can't think of one now. Also, what do you think about adding a method like 'addReview` which appends a review to the reviews list?